faq/public-gateways.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Public Gateways
+  description: Discover Public Gateways.
+content:
+  h1: Public Gateways
+  paragraph: Discover Public Gateways.
+hero: assets/public-gateways.webp
+---
+
+## Public Gateways
+
+### What is a Public Gateway?
+
+Public Gateways sit at the border of Private Networks. They provide services to automate the allocation of private IP addresses (DHCP) and deal with traffic entering and exiting the network (NAT).
+A Public Gateway can be attached to up to 8 Private Networks and up to 50 Public Gateways are supported per Organization.
+
+The Public Gateway can be configured through the console or the [API](https://developers.scaleway.com/en/products/vpc-gw/api/v1/).
+
+### Does the Public Gateway require a public IPv4 address?
+
+No. A public IPv4 address (aka. flexible IP) must be assigned to the Public Gateway at creation time, but you can detach it and delete it afterwards if you do not want to use the NAT feature.
+
+### Do I have to restart resources after attaching a Public Gateway to my Private Network?
+
+No. Resources in a Private Network for which autoconfiguration (DHCP) is enabled will automatically benefit from the services provided by the Public Gateway. 
+Autoconfiguration is automatically enabled in recent Instances. If your Instance is too old, you may have to update it or enable autoconfiguration by hand. For more details about configuring Instances, see [here](/network/public-gateways/how-to/configure-instances-public-gw/#how-to-carry-out-manual-configuration).
+
+### Can my Instances and other resources access the Internet without a public IPv4 address?
+
+Yes. With NAT enabled, the Public Gateway shares its public IPv4 address (aka. flexible IP) with the Instances and other resources attached to the Private Network, so that they can access the Internet.
+Moreover, the Public Gateway supports static NAT (aka. port forwarding), so that ingress traffic from the public Internet can reach Instances on the private network. This works by mapping pre-defined ports of the public IP address of the gateway to specific ports and IP addresses on the private network.
```

Comment:
```suggestion
Moreover, the Public Gateway supports static NAT (aka. port forwarding), so that ingress traffic from the public Internet can reach Instances on the Private Network. This works by mapping pre-defined ports of the public IP address of the gateway to specific ports and IP addresses on the Private Network.
```